docs/docsite/rst/aws_eda_guide.rst

```diff
@@ -0,0 +1,94 @@
+
+Rule #2: Re-create the trail
+----------------------------
+This rule handles the case when the trail is deleted. Basically, when these conditions `event.CloudTrailEvent.eventName=="DeleteTrail" and event.CloudTrailEvent.requestParameters.name==vars.cloudtrail_name` are met, the action to be takes implies running the `playbooks/eda/aws_restore_cloudtrail.yml` playbook. This playbook runs the Ansible Validated Content `cloud.aws_ops.awsconfig_multiregion_cloudtrail` role first which re-creates the trail and then the `cloud.aws_ops.enable_cloudtrail_encryption_with_kms` role to enable the encryption on the newly created trail.
+
+Rule #3: Cancels the deletion of the KMS key and re-enables it
+--------------------------------------------------------------
+This rule handles the case when the KMS key is deleted or disabled. This results in the condition `event.CloudTrailEvent.eventName=="ScheduleKeyDeletion" or event.CloudTrailEvent.eventName=="DisableKey"` that should be met to trigger this rule. When someone attempts to delete a KMS key intentionally or accidentally, a "ScheduleKeyDeletion" event is displayed in AWS CloudTrail. The KMS key is not deleted immediately; because deleting a KMS key is destructive and potentially dangerous, AWS KMS requires setting a 7-30 day waiting period. This situation is handled promptly by running `playbooks/eda/aws_restore_kms_key.yml` playbook which cancels the deletion of the KMS key. Similarly, when the KMS is disabled, the playbook reactivates it to restore the original state of the system.
+
+The playbook sets the KMS key ARN and uses it to determine whether to both cancel the KMS key deletion and to re-enable it.
+
+Follow the instructions defined in the README of [ansible/event-driven-ansible](https://github.com/ansible/event-driven-ansible#install) to install the collection and its dependencies.
+
+The rulebook is executed with the following command:
+
+```shell
+    ansible-rulebook --inventory inventory.yml --rulebook rulebooks/aws_manage_cloudtrail_encryption.yml --vars vars.yml
+```
+
+The inventory file used is like:
+
+```yaml
+  all:
+    hosts:
+    localhost:
+        ansible_python_interpreter: /path/to/python
+        ansible_connection: local
+```
+
+While, `vars.yml` file looks like:
+
+```yaml
+    _resource_prefix: ansible-cloudtrail-demo-eda
+    cloudtrail_name: "{{ _resource_prefix }}-trail"
+    s3_bucket_name: "{{ _resource_prefix }}-bucket"
+    kms_key_alias: "{{ _resource_prefix }}-key"
+    key_prefix: "{{ _resource_prefix }}"
```

Comment:
Can this be changed to s3_key_prefix ? With `key_prefix` ansible-lint throws [jinja[invalid]](https://github.com/redhat-cop/cloud.aws_ops/actions/runs/4722443875/jobs/8377027192?pr=33).